codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostStandardRequestDecoder.java

```diff
@@ -661,7 +662,7 @@ private static ByteBuf decodeAttribute(ByteBuf b, Charset charset) {
             return null; // nothing to decode
         }
 
-        ByteBuf buf = b.alloc().buffer(b.readableBytes());
+        ByteBuf buf = b.alloc().buffer(b.readableBytes(), b.readableBytes());
```

Comment:
I think the `b.readable()` for the second arg is not needed.

Reply:
Not mandatory, but there, it is valid (there is no need to allocate more than this in this specific case).
If you prefer, I could remove it (while valid).